docs/configuration.md

```diff
@@ -1493,6 +1493,14 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.streaming.stopGracefullyOnShutdown</code></td>
+  <td>false</td>
+  <td>
+    Since 1.4.0, Spark will stop StreamingContext without gracefully from JVM shutdown hook. Set it
```

Comment:
It sounds better. I'm not a native English speaker so I'll modify it as your suggestion.